.github/workflows/format-code.yml

```diff
@@ -1,19 +1,78 @@
-name: "Format Code: ensure code formatting guidelines are met"
+---
+name: 'Format Code: ensure code formatting guidelines are met'
+
 on:
-  pull_request:
-    types: [opened, edited, reopened, synchronize]
+  workflow_dispatch: null
+  schedule:
+    - cron: 45 4 * * 1-5
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  # Apply linter fixes configuration
+  APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
+  APPLY_FIXES_EVENT: all # Decide which event triggers application of fixes in a commit or a PR (pull_request, push, all)
+  APPLY_FIXES_MODE: commit # If APPLY_FIXES is used, defines if the fixes are directly committed (commit) or posted in a PR (pull_request)
 
-permissions: {}
+concurrency:
+  group: '${{ github.ref }}-${{ github.workflow }}'
+  cancel-in-progress: true
 
 jobs:
-  format-code:
-    permissions:
-      contents: write
+  build:
+    name: MegaLinter
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-      - uses: ministryofjustice/github-actions/code-formatter@15729e214b8e626847cf4b4596c7635ddb6b1b0a # v10
+      - name: Checkout Code
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
```

Comment:
keep the version number pls: # v3.3.0

Reply:
OK! All done in my most recent commit - [7b900a9](https://github.com/ministryofjustice/modernisation-platform/pull/3105/commits/7b900a94535a641690adfd05539e5df4da96219a)